packages/rolldown/src/cli/utils.ts

```diff
@@ -6,16 +6,44 @@ import type { ConfigExport } from '../types/config-export'
 /**
  * Console logger
  */
-export const logger = createConsola({
-  formatOptions: {
-    date: false,
-  },
-})
+export const logger = process.env.TEST
```

Comment:
Perhaps a more unique env identifier such as `ROLLDOWN_TEST` would be better since `TEST` is too common in development and may conflict with users' env

Reply:
nice advice.